docs/conf.py

```diff
@@ -42,8 +43,35 @@
     "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
     "sphinx.ext.autosectionlabel",
+    "sphinx.ext.inheritance_diagram",
 ]
 
+# Modules needed to import django files
+sys.modules["psycopg.types"] = Mock()
+# These modules run code on import which are dependant on other packages
+sys.modules["django.db.backends.postgresql.psycopg_any"] = Mock()
+sys.modules["django.contrib.gis.geos"] = Mock()
+sys.modules["django.contrib.gis.gdal.libgdal"] = Mock()
+sys.modules["django.contrib.gis.gdal.prototypes"] = Mock()
+sys.modules["django.contrib.gis.db.models.functions"] = Mock()
+# Load django apps for generating their source by 'sphinx.ext.viewcode'
+try:
+    from django.apps import apps
+    from django.conf import settings
+except (ImportError, ModuleNotFoundError):
+    pass
```

Comment:
Can you add please a message here since you are catching to exceptions.

Reply:
Thank you for this! 
Made me release it didn't make sense for this to not work and I was missing a dependency in the `docs/requirements.txt`
Made some updates :+1:

Reply:
@nessita do you think I should do something like this so existing contributors don’t have their docs build fail or do you think it’s better that it fails and they install the requirements? I could do a post on the discord? 🤔